src/moveit_opw_kinematics_plugin.cpp

```diff
@@ -274,11 +274,11 @@ void MoveItOPWKinematicsPlugin::expandIKSolutions(std::vector<std::vector<double
 }
 
 bool MoveItOPWKinematicsPlugin::searchPositionIK(const std::vector<geometry_msgs::Pose>& ik_poses,
-                                                 const std::vector<double>& ik_seed_state, double timeout,
-                                                 const std::vector<double>& consistency_limits,
+                                                 const std::vector<double>& ik_seed_state, double /*timeout*/,
```

Comment:
Interesting, I learned something new :) It seems like `robot_state.h` could also use this fix.